common/type.h

```diff
@@ -2453,6 +2453,19 @@ struct stats_t
 	uint64_t intersect_packets;
 	uint64_t unknown_network_type_packets;
 	uint64_t timeout_packets;
+
+	stats_t& operator+=(const stats_t& other)
+	{
+		current_count_packets += other.current_count_packets;
+		total_overflow_packets += other.total_overflow_packets;
+		not_fragment_packets += other.not_fragment_packets;
+		empty_packets += other.empty_packets;
+		flow_overflow_packets += other.flow_overflow_packets;
+		intersect_packets += other.intersect_packets;
+		unknown_network_type_packets += other.unknown_network_type_packets;
+		timeout_packets += other.timeout_packets;
+		return *this;
+	}
```

Comment:
I suggest moving this change into a separate commit. "Fragmentation untied from control and dataplane" does not seems like an appropriate place.